src/parsing/refactored_2_1/backups/final_updated_pipeline.py

```
@@ -13,9 +13,7 @@ async def ingest_sources(
 
     # Initialize components
     config = PipelineConfig(config_file)
-    embedding_manager = EmbeddingManager(
-        model=config.embedding_model, chunk_size=config.chunk_size
-    )
+    embedding_manager = EmbeddingManager(model=config.embedding_model, chunk_size=config.chunk_size)
```

Ruff (0.11.9)

Undefined name `EmbeddingManager`

(F821)